junit-platform-surefire-provider/src/test/java/org/junit/platform/surefire/provider/RunListenerAdapterTests.java

```diff
@@ -170,6 +171,8 @@ void notifiedWithParentDisplayNameWhenTestClassUnknown() throws Exception {
 	private static TestIdentifier newMethodIdentifier() throws Exception {
 		TestDescriptor testDescriptor = new MethodTestDescriptor(newId(), MyTestClass.class,
 			MyTestClass.class.getDeclaredMethod(MY_TEST_METHOD_NAME));
+
```

Comment:
I think the blank line here is more confusing than not, as I consider the 4 lines in this method to be part of the same "logical" block of code/functionality.